pkg/objstore/inmem/inmem.go

```diff
@@ -32,6 +32,11 @@ func (b *Bucket) Objects() map[string][]byte {
 	return b.objects
 }
 
+// Ensure that the bucket actually exists, if it doesn't it is created.
```

Comment:
It should start by `EnsureBucketExists ...` as this is an exported method. This applies to other comments as well.

Reply:
Done.